### PR TITLE
Add merge conflicted files to user info

### PR DESCRIPTION
**Pull request by pietbrauer**
This the add on to https://github.com/libgit2/objective-git/pull/538#issuecomment-175405892

### CODE REVIEW

File: ObjectiveGit/GTRepository+Pull.m [hunk 1]
```diff
@@ -20,6 +20,8 @@
 #import "GTIndexEntry.h"
 #import "git2/errors.h"
 
+NSString * const GTPullMergeConflictedFiles = @"GTPullMergeConflictedFiles";
```
Comment (joshaber) [inline ObjectiveGit/GTRepository+Pull.m]: We should expose this from the header too, so users don't have to know the string literal.

**Comment by pietbrauer** [comment]
Done :+1:

File: ObjectiveGit/GTRepository+Pull.h [hunk 2]
```diff
@@ -11,6 +11,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+extern NSString * const GTPullMergeConflictedFiles;
```
Comment (joshaber) [inline ObjectiveGit/GTRepository+Pull.h]: Documentation please :grimacing:

**Comment by joshaber** [comment]
Just :one: :memo:

**Comment by pietbrauer** [comment]
Done :white_check_mark:

**Comment by joshaber** [comment]
:ship:

**Pull request merged**